plugins/ssl/letsencrypt_weekly

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+: << =cut
+=head1 NAME
+
+letsencrypt_weekly - monitor the number of CSRs by week for /etc/letsencrypt/csr/
+
+see https://letsencrypt.org/docs/rate-limits/
+
+= head1 CONFIGURATION
+
+There are no special configuration options for this plugin
+
+=head1 Dependencies
+
+Dependencies: openssl
+
+=head1 AUTHOR
+
+andreas perhab - andreas.perhab@wt-io-it.at
+https://www.wt-io-it.at/
+
+=head1 LICENSE
+
+GPLv2
+
+=head1 MAGIC MARKERS
+
+ #%# family=auto
+ #%# capabilities=autoconf
+
+=cut
+
+. "$MUNIN_LIBDIR/plugins/plugin.sh"
+
+warning=${warning:-:40}
+
+if [ "$1" = "autoconf" ] ; then
+    find /etc/letsencrypt/csr/ > /dev/null 2>&1 && echo "yes" || echo "no (no files below /etc/letsencrypt/csr)"
+elif [ "$1" = "config" ] ; then
+	echo "graph_title Letsencrypt certificate requests during last week"
+	echo "graph_args --base 1000"
+	echo "graph_vlabel Number of certificates"
+	echo "graph_category security"
+	echo "letsencrypt_weekly.label Letsencrypt certificates last week"
+	print_warning "letsencrypt_weekly"
+	echo "letsencrypt_weekly.critical :50" #letsencrypt doesn't allow more than 50 certificates per week
```

Comment:
Regarding the hard limit of `50`: I would suggest, to make this configurable, too. `letsencrypt` will surely change the rules from time to time. It would be nice, if people could adjust to such changes via configuration and not via code.

Reply:
i added a variable (and config description in the doc)